package.json

```diff
@@ -7,9 +7,9 @@
     "@rails/actioncable": "^6.1.5",
     "@rails/activestorage": "^6.1.5",
     "@rails/ujs": "^6.1.5",
-    "@rails/webpacker": "^5.4.3",
+    "@rails/webpacker": "5.4.3",
     "ansi-regex": "^6.0.1",
-    "bootstrap": "^5.1.3",
+    "bootstrap": "^4",
```

Comment:
I think this did not change the version. The syntax probably means "at least 4 or any version above that".

It would probably be better to use the latest 4.x version number instead:

```js
"bootstrap", "^4.6.1",
```